script.js

```diff
@@ -1,24 +1,48 @@
-document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Only add DOMContent once in JS and whatever JS code goes inside the DOMContent JS.